app/models/application_record.rb

```diff
@@ -1,3 +1,12 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
+
+  def tpying
+    puts 'esential typo'
+  end
+
+
+  def esential
```

Comment:
__[Misspell]__

"esential" is a misspelling of "essential"


<!-- Comment by [Sider](https://sider.review) -->